### PR TITLE
The Great Materials Revert

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
Reverts our Material Outputs and Lathe Production Costs to pre-Citadel balance change levels, due to public response.
The people have spoken!
image

## Why It's Good For The Game
_Change is bad etc. etc. unga bunga._
In all seriousness, this was a pretty drastic change that nobody wanted except the maintainers over at Citadel, who seem to have very skewed ideas of what is "fun" compared to us.
In layman's terms: _We never asked for this._

## Changelog
:cl:
balance: reverted our Lathe-Costs and Material-Outputs to pre-citadel nerfs or atleast close to them.
/:cl:

### CODE REVIEW

**Comment by ShadeAware** [comment]
### Ignore all of the commit fuckery near the beginning there, my branch is very old and strange.

**Comment by RatFromTheJungle** [comment]
back to how it was 'before' isn't the best point to go, especially when your 'balance vote' (which is never a good indicator because a majority vote is never a good idea for balance changes) had just a yes/no option, with no middle-ground, meaning it was either 'how citadel made it' or 'everything costs essentially nothing'.

**Comment by improvedname** [comment]
Agreed
I disagreed with the orm multiplier removal.
But the protolathe tier halfing was a great change because the cost halved eacj time you upgrade it so a normal boh costs 1500 diamonds at t4(which is a a diamond) otherwise the orginal price was 12000 diamonds(which is 6 diamonds at t1) but with the orm multiplier you will have plenty of materials

**Comment by ShadeAware** [comment]
This doesn't seem to be going anywhere; closing unless otherwise asked to re-open.
